src/rebar_mix_protocol_consolidation.exs

```diff
@@ -1,6 +1,6 @@
 # rebar3 provider sets this to all deps of the project being built
-paths = String.split(System.get_env("REBAR_DEPS_EBIN"), ":")
-out_dir = System.get_env("REBAR_PROTOCOLS_OUTDIR")
+paths = String.split(System.get_env("REBAR_DEPS_EBIN", ""), ":")
```

Comment:
This is throwing, for me, with the env. var being `nil` in this context. I didn't look into `rebar_utils:` closely enough, yet, to understand how this can be improved.